src/core/lib/gprpp/status_helper.h

```diff
@@ -38,7 +38,7 @@ struct google_rpc_Status;
 struct upb_Arena;
 }
 
-#define RETURN_IF_ERROR(expr)           \
```

Comment:
Let's change this to `GRPC_RETURN_IF_NOT_OK` - we should in general keep things in our own namespace